torch/nn/modules/module.py

```diff
@@ -105,19 +106,23 @@ def forward(self, *input):
         """
         raise NotImplementedError
 
-    def register_buffer(self, name, tensor):
-        r"""Adds a persistent buffer to the module.
+    def register_buffer(self, name, tensor, persistent=True):
+        r"""Adds a buffer to the module.
 
         This is typically used to register a buffer that should not to be
         considered a model parameter. For example, BatchNorm's ``running_mean``
-        is not a parameter, but is part of the persistent state.
+        is not a parameter, but is part of the module's state. Buffers, by
+        default, are persistent and will be saved alongside parameters. This
```

Comment:
Could you make this clearer that the only difference here is that non persistent buffers won't show up in the state dict load/save while the persistent ones will.
There is no other difference right?

Reply:
Done. Yes, there's no other difference.